src/Classes/DbConnector.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Furniture\Classes;
+
+use \PDO;
+
+class DbConnector
+{
+    private static PDO $db;
+    const HOST = 'db';
+    const DB = 'furniture';
+    const USER = 'root';
+    const PASSWORD = 'password';
+
+    const OPTIONS = [
+        PDO::ATTR_DEFAULT_FETCH_MODE => PDO::FETCH_ASSOC,
+        PDO::ATTR_ERRMODE => PDO::ERRMODE_EXCEPTION,
+    ];
+
+    public static function connectToDb(): PDO
```

Comment:
Since this should always return a DB connection (and only connect if it's the first time), I would call it something like `getDbConnection()`